sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapi/AADOAuth2AuthorizedClientCredentialRepositoryTest.java

```diff
@@ -48,81 +45,68 @@
             + "R-s_LOz1eBiMboKWIPiQZwWQ-peGNsalNriss4_x4pwOwYlMqeZJBk1tIyON0nYNLTU169_KSGHIlTtVtaAlNnt9C2Ajg1PTvJvj3fsg"
             + "FhZpRbO4XBs6nEjFSwPC0RII36raH9wjgveNn63LPg";
 
+    private ClientRegistration fake;
+    private Authentication principal;
+    private OAuth2AuthorizedClient oAuth2AuthorizedClient;
     private InMemoryClientRegistrationRepository clientRegistrationsRepo;
-    private OAuth2AuthorizedClient authorizedClient;
     private AADResourceServerOAuth2AuthorizedClientRepository authorizedRepo;
-    private JwtAuthenticationToken jwtAuthenticationToken;
-    private MockHttpServletRequest mockHttpServletRequest;
+    private InMemoryOAuth2AuthorizedClientService inMemoryOAuth2AuthorizedClientService;
+    private HttpServletRequest request = mock(MockHttpServletRequest.class);
```

Comment:
for mockobject, I think we should better name the variable name begin with “mock”
eg：request--> mockRequest
to make it clear